_commands/basics/kill.md

```diff
@@ -3,22 +3,51 @@
 
 kill
 --
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
 
-<!-- minimal example -->
+`kill` is used to send a signal to a process.
+
 ~~~ bash
-*short* example of command usage and output
+$kkill [-s] [-l] PID(s)
 ~~~
 
+--signal: Specify the signal to be sent.
```

Comment:
Should this be `-s` here instead of `--signal` since in the example above you used `-s` and `-l`?

It should also be surrounded with backticks to render it as code so "-s" -> `-s`